jupyterhub/oauth/provider.py

```diff
@@ -342,7 +342,7 @@ def save_bearer_token(self, token, request, *args, **kwargs):
         orm_access_token = orm.OAuthAccessToken(
             client=client,
             grant_type=orm.GrantType.authorization_code,
-            expires_at=orm.OAuthAccessToken.now() + token['expires_in'],
+            expires_at=(orm.OAuthAccessToken.now() + token['expires_in']),
```

Comment:
Hmmm I would guess this is exactly the same as not having (), can you motivate this technically?

Reply:
You're right, the code is incomplete. It must have been:

expires_at=int(orm.OAuthAccessToken.now() + token['expires_in']),

- Just a basic float to int conversion.

The fix started as a RUN instruction with a sed regexp on top of the official docker image, and then I tried to replicate the change in this PR. 13 days long workweek didn't play well for me as, apparently, I didn't see what I was copying. Should have left it till Monday to be more attentive. Such a shame. Sorry for that. 😅

Reply:
@consideRatio I've just added the missing `int`, should be fine now :)